### PR TITLE
Configuration ezfix

### PR DESCRIPTION
**Pull request by Amir974**
#6385 related

**Description:**

## Checklist:

- [x] Branch: `current`
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.acer_projector.markdown [hunk 1]
```diff
@@ -27,10 +27,21 @@ switch:
...
+  required: true
+  type: string
+name:
+  description: The name to use when displaying this switch.
+  required: false
+  type: string
+timeout:
+  description: Timeout for the connection in seconds.
+  required: false
+  type: int
```
Comment (WofWca) [inline source/_components/switch.acer_projector.markdown]: Integer

File: source/_components/switch.acer_projector.markdown [hunk 2]
```diff
@@ -27,10 +27,21 @@ switch:
...
+write_timeout:
+  description: Write timeout in seconds.
+  required: false
+  type: int
```
Comment (WofWca) [inline source/_components/switch.acer_projector.markdown]: Integer

**Review by WofWca — CHANGES_REQUESTED** [review]

**Comment by Amir974** [comment]
https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

States "int" is acceptable value for type (not integer) - a bit confusing
Where can I find a full list of types ?

**Comment by WofWca** [comment]
It's been worked on in the recent time. Some changes didn't even go live. Here's [the GitHub page](https://github.com/home-assistant/developers.home-assistant/blob/master/docs/documentation_create_page.md).

**Comment by frenck** [comment]
@Amir974, @WofWca is correct, we've changed the docs a bit, based on recent feedback.
The latest version of the documentation can be found here:

https://developers.home-assistant.io/docs/en/next/documentation_create_page.html#configuration

The "go live" for the developers' documentation is in sync with the Home Assistant releases.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thank you @Amir974!

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**